service/src/main/java/org/broadinstitute/listener/config/CorsSupportProperties.java

```diff
@@ -1,4 +1,10 @@
 package org.broadinstitute.listener.config;
 
+import java.util.List;
+
 public record CorsSupportProperties(
-    String preflightMethods, String allowHeaders, String maxAge, String contentSecurityPolicy) {}
+    String preflightMethods,
+    String allowHeaders,
+    String maxAge,
+    String contentSecurityPolicy,
+    List<String> validHosts) {}
```

Comment:
Where is the allowlist of valid hosts defined? Is it in the yaml file below?

Reply:
This is defined in Leonardo, and is actually added to the listener start command. We also add the SERVER_APP_WEBSOCKET_HOST which is generated some time during VM creation.

https://github.com/DataBiosphere/leonardo/pull/3311/files#diff-375ff112a6898504a9074d258d167ba63d6300124c1438b0351561eb7b8fc4ecR242

Reply:
Ah that makes a lot of sense, thanks!